src/utils/api/block/index.test.js

```diff
@@ -0,0 +1,20 @@
+import * as block from './index';
+import { subscribe, unsubscribe } from '../ws';
+
+jest.mock('../ws');
+
+describe('Block api module', () => {
+  it('Should call ws subscribe with parameters', () => {
+    const serviceUrl = 'http://sample-service-url.com';
+    block.blockSubscribe({ serviceUrl });
+    expect(subscribe).toHaveBeenCalledTimes(1);
+    expect(subscribe.mock.calls[0][0]).toContain(serviceUrl);
+    expect(subscribe.mock.calls[0][1]).toContain('blocks/change');
```

Comment:
Same here, Jest shorthands are more readable and therefore the real goal of the test is easier to detect with them. At the first look, it's hard to say what exactly `subscribe.mock.calls[0][0]` and `subscribe.mock.calls[0][1]` are.